File: AMS_电子对账/code/client/source/YssElecReco/Service/Er/IErZcfzbService.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Linq;
using System.Reflection;
using System.Windows.Forms;
using YssElecReco.pojo.Er;
using FAST.Common.Service.Services.Base;
using FAST.Common.Service.Attributes;
using YssElecReco.Context;

namespace YssElecReco.Service.Er
{
    /// <summary>
    /// 浏览电子对账信息service
    /// </summary>
    [ServiceAttribute(ServiceId = ElecrecoServiceIdConstant.OSGI_ELECRECO, ServiceController = "com.yss.uco.elecreco.er.erzcfzb.controller.IErZcfzbServiceController")]
    public interface IErZcfzbService : IServiceBus
    {
        /// <summary>
        /// 根据添加查询电子对账资产负债信息
        /// </summary>
        /// <param name="paraDict">paraDict</param>
        /// <returns>返回资产负债列表</returns>
        [MethodAttribute]
        List<ErZcfzb> getZcfzData(Dictionary<string, string> paraDict);
    }
}

File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/erzcfzb/service/IErZcfzbService.java
package com.yss.uco.elecreco.er.erzcfzb.service;

import java.util.HashMap;
import java.util.List;

import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;
import com.yss.uco.elecreco.er.erzcfzb.pojo.ErZcfzb;

@RestfulSupported
public interface IErZcfzbService extends IServiceBus {

	/**
	 * 根据条件查询电子对账资产负债信息
	 * @param paraMap
	 * @return
	 */
	 public List<ErZcfzb> getZcfzData(HashMap<String, Object> paraMap);
}
